static/app/components/onboardingWizard/task.tsx

```diff
@@ -65,6 +66,16 @@ function Task(props: Props) {
     recordAnalytics(task, organization, 'clickthrough');
     e.stopPropagation();
 
+    switch (task.task) {
```

Comment:
@roggenkemper can we move this logic to a function in `DemoWalkthroughStore`? I think that would be a log cleaner.